MembraneRTC/src/main/java/org/membraneframework/rtc/transport/PhoenixTransport.kt

```diff
@@ -7,14 +7,16 @@ import org.membraneframework.rtc.events.ReceivableEvent
 import org.membraneframework.rtc.events.SendableEvent
 import org.membraneframework.rtc.events.serializeToMap
 import org.membraneframework.rtc.utils.ClosableCoroutineScope
+import org.membraneframework.rtc.utils.SocketConnectionParams
 import org.phoenixframework.Channel
 import org.phoenixframework.Socket
 import timber.log.Timber
 
 public class PhoenixTransport constructor(
-    val url: String,
-    val topic: String,
-    private val ioDispatcher: CoroutineDispatcher
+    private val url: String,
+    private val topic: String,
+    private val ioDispatcher: CoroutineDispatcher,
+    private val params: SocketConnectionParams? = null
```

Comment:
One more thing, can we have `emptyMap()` instead `null`? We don't want to break backward compatibility if it's not necessary

Reply:
done!